### PR TITLE
Incorrect namespace

### PR DESCRIPTION
**Pull request by Thoulah**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | -

### CODE REVIEW

File: src/PhpFile.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Yiisoft\I18n\Resource;
+namespace Yiisoft\I18n;
```
Comment (samdark) [inline src/PhpFile.php:3]: That should be `Yiisoft\I18n\Message`, I think.

Comment (Thoulah) [reply src/PhpFile.php:3]: Not with current setup. https://github.com/yiisoft/i18n-message-php/blob/master/composer.json#L29

Comment (samdark) [reply src/PhpFile.php:3]: Yes, should be adjusted as well.

Comment (samdark) [reply src/PhpFile.php:3]: https://github.com/yiisoft/docs/blob/master/004-namespaces.md

Comment (Thoulah) [reply src/PhpFile.php]: You want to rename from `PhpFile` to `Php` as well?

Comment (samdark) [reply src/PhpFile.php]: No.

Comment (Thoulah) [reply src/PhpFile.php]: You got it. Namespace is now Message

**Comment by samdark** [comment]
Thanks.

**Pull request merged**